client/src/views/viewers/AudioViewer.vue

```diff
@@ -95,6 +97,10 @@ function updateMediaData(event: Event): void {
 //       return pause;
 //   }
 // }
+
+function toggleFullScreen(): void {
+  audioElement.value?.requestFullscreen();
```

Comment:
> The Element.requestFullscreen() method issues an asynchronous request (https://developer.mozilla.org/en-US/docs/Web/API/Element/requestFullscreen)

So all those functions should be async, and the call to `requestFullscreen` should be awaited. Valid for all calls in all viewers.